products/nvidia-driver.md

```diff
@@ -13,39 +13,57 @@ LTSLabel: "<abbr title='Long Term Support Branch'>LTSB</abbr>"
 eoasColumn: true
 releaseDateColumn: true
 
+# Branch type see https://docs.nvidia.com/datacenter/tesla/drivers/releases.json
+#
 # eoas(x) is:
-# - false for NFB releases
+# - true for NFB releases
 # - releaseDate(x) + 1 year for PB and LTS releases
 #
 # eol(x) is:
 # - releaseDate(x) + 1 year for NFB and PB releases
 # - releaseDate(x) + 3 years for LTS releases
 releases:
--   releaseCycle: "R565-Linux (NFB)"
+-   releaseCycle: "R570-Linux (PB)"
+    releaseDate: 2025-01-27
+    eoas: 2026-01-27
+    eol: 2026-01-27
+    latest: "570.86.15"
+    latestReleaseDate: 2025-01-27
+    link: https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-570-86-15/
+          
+-   releaseCycle: "R570-Windows (PB)"
+    releaseDate: 2025-01-27
+    eoas: 2026-01-27
+    eol: 2026-01-27
```

Comment:
same for different platform versions

Reply:
Updated 8d5279e699dd2f39317087dbd99c51f7c278d881